maintenance/pin-helper.sh

```diff
@@ -13,7 +13,10 @@ pip install --upgrade pip
 pip install pip-compile-multi
 
 apt-get update
-apt-get install -y libsodium-dev
+apt-get install -y \
+    gir1.2-ostree-1.0 \
+    libgirepository1.0-dev \
+    libsodium-dev
```

Comment:
do we still need `libsodium-dev`? who's using this?

Reply:
This script is used by all docker images, not only by the new one. IIRC it's used by pushsnap.

Reply:
Right, forgot of that, makes sense.